v2/common/src/main/java/com/google/cloud/teleport/v2/utils/BeamSchemaUtils.java

```diff
@@ -100,17 +100,27 @@ private static Schema fromJson(JsonParser jsonParser) throws IOException, Schema
   private static List<Field> getFieldsfromJsonNode(JsonNode jsonNode)
       throws SchemaParseException {
     List<Field> fields = new LinkedList<>();
+    Field field;
     for (JsonNode node : jsonNode) {
       if (!node.isObject()) {
         throw new SchemaParseException("Node must be object: " + node.toString());
       }
       String type = getText(node, FIELD_TYPE, "type is missed");
       String name = getText(node, FIELD_NAME, "name is missed");
-      fields.add(Field.of(name, stringToFieldType(type)));
+      boolean nullable = getOptionalBoolean(node, FIELD_NULLABLE, false);
```

Comment:
let's make the default value definition more explisitly. You could move it to private final var